Tests/UnitTests/Scripts/tests.ts

```diff
@@ -1,6 +1,14 @@
-﻿mocha.setup({ ui: "bdd", reporter: "spec", retries: 5 });
+﻿import MochaDefault from "mocha";
+const Mocha = MochaDefault as typeof import("mocha");
```

Comment:
Is this equivalent to:
```suggestion
﻿import * as MochaDefault from "mocha";
```